pkg/virt-controller/watch/migration.go

```diff
@@ -1407,6 +1400,43 @@ func (c *MigrationController) getNodeForVMI(vmi *virtv1.VirtualMachineInstance)
 	return node, nil
 }
 
+func (c *MigrationController) alertIfHostModelIsUnschedulable(vmi *virtv1.VirtualMachineInstance, targetPod *k8sv1.Pod) {
+	const warningMsgPattern = "Migration cannot proceed since no node is suitable to run the required CPU model / required features: %v"
```

Comment:
I tend to disagree :)

This is relevant to a very specific scope, so I don't see a reason to get it out to a broader scope. In general I think this should be the approach - variables need to be declared at the most inner scope they're relevant to. This way anyone who reads the code has a clear idea of what this variable affects (and what it can't affect).